aether/sdk/conf/settings.py

```diff
@@ -559,10 +559,14 @@ def get_required(name):
         AWS_S3_REGION_NAME = get_required('AWS_S3_REGION_NAME')
         AWS_DEFAULT_ACL = get_required('AWS_DEFAULT_ACL')
 
+        STATICFILES_STORAGE = 'storages.backends.s3boto3.S3Boto3Storage'
+
     elif DJANGO_STORAGE_BACKEND == 'gcs':
         INSTALLED_APPS += ['storages', ]
         DEFAULT_FILE_STORAGE = 'storages.backends.gcloud.GoogleCloudStorage'
 
+        STATICFILES_STORAGE = 'storages.backends.gcloud.GoogleCloudStorage'
```

Comment:
Take in mind that `settings.STATICFILES_STORAGE` would raise an error even if you check before `if settings.STORAGE_REQUIRED:` because minio doesn't declare this variable. :grimacing:

Reply:
This is used only required when `DJANGO_STORAGE_BACKEND ` matches `s3 || gsc`

Reply:
Minio can work as a GCS or S· wrapper (_one library to rule them all_ :ring: )

Reply:
Are you saying we shouldn't use `django-storages`?

Reply:
No, I'm saying that minio is another option to serve static files as GCS and S3. We are using minio to serve our local files using the file system, but we could also connect minio to S3 and the behavior is exactly the same, and I don't see why we should exclude it in this case.

Reply:
https://django-minio-storage.readthedocs.io/en/latest/usage/

Reply:
Am not sure i understand what you mean; do you want minio to use static files collected on `s3` or `gcs`?

Reply:
I want you to add in the minio side:

```python
    if DJANGO_STORAGE_BACKEND == 'minio':
        # ...
        STATICFILES_STORAGE = 'minio_storage.storage.MinioStaticStorage'
        # ...
```

Reply:
I would also appreciate if there is another option to decide if the static content is pushed to the storage or not :woman_shrugging:

Reply:
collecting static files on minio will have to come with proxying which is what we are trying to reduce with a CDN solution

Reply:
But this is an opensource module, you can not restrict the features to our current needs... if it's valid for one, it's valid for all... we can decide to use or not the minio option but it must be there along with the others.